openid-4-verifiable-presentations-1_0.md

```diff
@@ -261,6 +261,15 @@ This specification defines the following new parameters:
 
 A public key to be used by the Wallet as an input to the key agreement to encrypt Authorization Response (see (#jarm)). It MAY be passed by the Verifier using the `jwks` or the `jwks_uri` claim within the `client_metadata` or `client_metadata_uri` request parameter.
 
+`request_uri_method`: 
+: OPTIONAL. A string determining the HTTP method to be used with the `request_uri` passed in the same request. This specification defines two values for this parameter: `GET` and `POST`. `GET` is 
+the mode as defined by in [@RFC9101], where the Wallet sends a GET request to fetch a request object. `POST` is a new mode defined in this specification where the Wallet requests the 
+creation of a fresh request object as defined in [@RFC9101] by sending a POST request to the request URI. The details of this request are defined in (#request_uri_method_post). 
+`request_uri_mode` MUST only be present if the request also contains a `request_uri` parameter. If the parameter is not present, the Wallet MUST process the `request_uri` as defined in 
```

Comment:
```suggestion
`request_uri_mode` REQUIRED when the request contains a `request_uri` parameter. If `request_uri_mode` is not present, the Wallet MUST process the `request_uri` as defined in 
```

Reply:
it seems to me that request_uri_mode is deprecated and that we can use request_uri_method only.

@tlodderstedt if you confirm this we can remove this parameter and update the next text and examples

Reply:
`request_uri_mode` is replaced by `request_uri_method`

Reply:
fixed example